src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -1872,6 +1905,38 @@ unsafe public static IntPtr StringToCoTaskMemUni(String s)
         }
 
         [System.Security.SecurityCritical]  // auto-generated_required
+        unsafe public static IntPtr StringToAllocatedMemoryUTF8(String s)
```

Comment:
Unfortunately, given that we no longer have the Pinvoke contract, we need to name this StringToCoTaskMemUTF8.